wallet/concepts/sdk.md

```diff
@@ -1,13 +1,13 @@
 ---
 description: Learn about MetaMask SDK.
-sidebar_position: 1
+sidebar_position: 2
 ---
 
 # MetaMask SDK
 
 MetaMask SDK is a library that provides a reliable, secure, and seamless connection from your dapp
 to the MetaMask browser extension and MetaMask Mobile.
-You can install the SDK on existing dapps, and call any [provider API](../reference/provider-api.md) methods from
+You can install the SDK on existing dapps, and call any [provider API](provider-api.md) methods from
```

Comment:
"on existing dapps" didn't read right. So suggesting "You can install the SDK for existing dapps,"